jerry-core/ecma/base/ecma-helpers-string.cpp

```diff
@@ -1286,6 +1286,64 @@ ecma_compare_ecma_strings_relational (const ecma_string_t *string1_p, /**< ecma-
 } /* ecma_compare_ecma_strings_relational */
 
 /**
+ * Lengths for numeric string values
+ */
+static const uint32_t nums_with_ascending_length[] =
+{
+  1u,
+  10u,
+  100u,
+  1000u,
+  10000u,
+  100000u,
+  1000000u,
+  10000000u,
+  100000000u,
+  1000000000u
+};
+
+/**
+ * Maximum length of numeric strings
+ */
+static const uint32_t max_uint32_len = sizeof (nums_with_ascending_length) / sizeof (uint32_t);
+
+/**
+ * Get size of container heap number of ecma-string
+ *
+ * Note: the number size and length are equal
+ *
+ * @return number of bytes in the buffer
+ */
+static lit_utf8_size_t
+ecma_string_get_heap_number_size (mem_cpointer_t number_cp) /**< ecma-string */
+{
+  const ecma_number_t *num_p = ECMA_GET_NON_NULL_POINTER (ecma_number_t, number_cp);
+  lit_utf8_byte_t buffer[ECMA_MAX_CHARS_IN_STRINGIFIED_NUMBER];
+
+  return ecma_number_to_utf8_string (*num_p, buffer, sizeof (buffer));
+} /* ecma_string_get_heap_number_size */
+
+
+/**
+ * Get size of the number stored locally in the string's descriptor
+ *
+ * Note: the represented number size and length are equal
+ *
+ * @return size in bytes
+ */
+static ecma_length_t
+ecma_string_get_number_in_desc_size (const uint32_t uint32_number)
```

Comment:
Please add comment on argument.
moreover, I'd swap the order of these two new helper functions to get this one closer to the data it uses (`nums_with_ascending_length` and `max_uint32_len`).